tests/integrationtest/t/privilege/privileges.test

```diff
@@ -237,7 +234,7 @@ GRANT ALL ON *.* TO 'hasgrant';
 GRANT ALL ON mysql.* TO 'withoutgrant';
 connect (hasgrant,localhost,hasgrant,,);
 connection hasgrant;
---error WITHOUT GRANT OPTION
+--error 8121
```

Comment:
Just a tip, one can also use named errors, both the golang constant and the MySQL errno name.